pkg/controller/servicemesh/memberroll/controller.go

```diff
@@ -413,6 +414,13 @@ func (r *MemberRollReconciler) reconcileObject(ctx context.Context, roll *maistr
 	} else if kialiErr != nil {
 		return reconcile.Result{}, kialiErr
 	}
+	if mesh != nil {
+		meshMode, err := getMeshMode(mesh)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
+		internalmetrics.GetMemberCounter(mesh.GetNamespace(), mesh.Spec.Version, meshMode).Set(float64(len(roll.Status.Members)))
```

Comment:
We may need to consider whether using `roll.status.configuredMembers` makes more sense, because `roll.status.members` includes namespaces that don't exist (but are listed in `spec.members`), whereas `configuredMembers` contains only those namespaces that are actually part of the mesh.

Reply:
we might just want to measure both - configured_members and specified_members maybe?

Reply:
IMO we should get only the namespaces that are actually part of the mesh. It would be a better "picture" of the mesh